### DISCUSSION TITLE
Is it possible to tell an OverlayContainer to render the overlay at the end of an `iframe`'s `document`?

### DISCUSSION THREAD

**Discussion by stefanprobst** [adobe/react-spectrum#2211] [Q&A] [answered]
it seems that currently the `div[data-overlay-container]` will always be appended to the global `document`. is there a way to append it to an `iframe`'s `document` (e.g. in a CMS preview iframe)?

**Comment by snowystinger** [comment, marked as answer]
If you're in the iframe, then that's where it'll be appended, you can't render anything outside. If you're outside, then you can't render anything inside. That's the point of an iframe. If you control both though, you could send a post message to tell the iframe to render something in itself.
I think maybe you meant something different though and what you're looking for is this https://github.com/adobe/react-spectrum/blob/ab3c35039671bb0e58a3e5b2e01d2f7d6585d2c8/packages/%40react-aria/overlays/src/useModal.tsx#L114

**Reply by stefanprobst** [reply]
ah yes, `portalContainer` was what i was looking for! thanks!
